crates/burn-tensor/src/tensor/data.rs

```diff
@@ -29,7 +29,7 @@ pub enum DataError {
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct TensorData {
     /// The values of the tensor (as bytes).
-    value: Vec<u8>,
+    pub value: Vec<u8>,
```

Comment:
I think we should rename that field to `bytes`, it's more accurate than value.

Reply:
Done.